outsettingsvalues/tests.py

```diff
@@ -0,0 +1,82 @@
+"""
+Test outsettingsvalues view.
+"""
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.shortcuts import reverse
+from django.test import Client, RequestFactory, TestCase
+
+from outsettingsvalues.views import BaseView
+
+
+class TestBaseView(TestCase):
```

Comment:
Could you use mock object here to take control of the environment for the test(get expected result, which not depend on which platform it runs)